README.md

```diff
@@ -1,39 +1,82 @@
+  - item[n].type
+     - example: "choice", "decimal", "string", "display"
      - REQUIRED.
      - Note: we ignore other directives eg item.extension.valueCodeableConcept.coding.code "drop-down".  
+  - item[n].answerOption[n]
+    - item.answerOption[n].valueCoding
+      - code: QuestionnaireResponse refers to this
+        - This is not used for computed mappings back *to* the non-FHIR source (eg dhair2); that source maps in this direction (eg dhair2's options.fhir_code).
+        - If we identify a FHIR resource for the Questionnaire that's established in the community (usually loinc), then we'll use its values for these. Otherwise, use whatever is easy... a pattern like option.id from dhair2 eg [this PainTracker body diagram question](https://github.com/uwcirg/fhir-questionnaires/blob/main/CIRG-PainTracker-Location-Body-Diagram.json#L17)' is good, but sometimes not easy (hard to scale).
+      - display: the text displayed for the option.
+    - item.answerOption[n].extension
+      - "url": "http://hl7.org/fhir/StructureDefinition/ordinalValue"
+        - almost always valueDecimal; rarely valueString (example [here](https://github.com/uwcirg/asbi-screening-app/blob/master/src/fhir/1_Questionnaire-C-IDAS.json)).
+      - "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-optionPrefix" I see this is nearly always populated, but Amy do you use this at all? Seems redundant w/ ordinalValue (just above).
+  - item[n]._text.extension where item[n].type = "display"
+    - XTHML displayed...
+      - ... during the questionnaire, usually instructions.
+      - ... sometimes in the [summary report](https://github.com/uwcirg/patient-summary) "about" pop-up, but as of 2023-10-12 we are moving to `description` for that.
+      - Not a question (does not have an answerOption[]).
+      - Usually at item[0] (the screener gives it its own page in this case)
+      - Sometimes later as instructions for a specific question item
+      - Examples of both in [MINICOG in the questionnaire filler](https://github.com/uwcirg/asbi-screening-app/blob/master/src/fhir/1_Questionnaire-MINICOG.json).
+    - "url": "http://hl7.org/fhir/StructureDefinition/rendering-xhtml"
+    - "valueString": XTHML
+    - Examples
+      - CIRG-PC-PTSD-5.json
+      - CIRG-PHQ-4.json
+      - 1_Questionnaire-USAUDIT.json (screener app)
+  - item[n]."extension"."valueCoding"
+    - We sometimes use this to indicate that an item is a score [here](https://github.com/uwcirg/fhir-questionnaires/pull/2/files#diff-66fd6a93556a044e8ffa3a290dac3e49b37b29b60c0cdddfb2645fe5cea49ae2R582)
+    - Amy do you read this for anything?
+  - item[n]."enableWhen"
+    - Amy you're not using this for anything, are you? I see it for the DCW Audit questionnaires, but we don't use those...
```

Comment:
@mcjustin this is used by ASBI questionnaire-to-survey tool to determine whether to show a question, see [here](https://github.com/asbi-cds-tools/questionnaire-to-survey/blob/63f117fd666830f4fdecec10ad4cea363abed413/fhirConversionTools.js#L346)

Reply:
thanks, I've just edited to reflect this.